### ISSUE TITLE
Feature Request: Build appimages on bionic

### ISSUE THREAD

**Issue by theofficialgman** [smartavionics/Cura#175]
**Is your feature request related to a problem? Please describe.**
Unable to run on Ubuntu bionic (eg: nvidia jetson devices)

**Describe the solution you'd like**
Build the arm64 (and armhf) appimages on ubuntu bionic to avoid GLIBC 2.28 being necessary from building on buster.

**Comment by smartavionics** [comment]
Hi, is it straightforward to install glibc 2.28 on bionic? Does doing that cause any other issues? If not, then it would be better to stay as we are now so that people who are still running buster can use the appimages.

**Comment by theofficialgman** [comment]
> Hi, is it straightforward to install glibc 2.28 on bionic? Does doing that cause any other issues? If not, then it would be better to stay as we are now so that people who are still running buster can use the appimages.

No, attempting to install glibc 2.28 will break many other packages on bionic. That's basically the most damaging thing you can do to an OS. Glibc is the base c library that every single package in the os depends on.

Building on bionic shouldn't cause any issues with using the appimage on buster (or bullseye which many people do).

**Comment by smartavionics** [comment]
Yes, I can see now looking at the specs for the various OS releases that the only real solution is to build a release on a bionic system. I would be happy to do that but, unfortunately, I don't have it installed on any of my computers. Does it run on the Pi? If so, I could install it on a Pi 4. Do you have a link for an installation image?

**Comment by theofficialgman** [comment]
There are 18.04 server images here for the pi
https://wiki.ubuntu.com/ARM/RaspberryPi
You can install the desktop on that with accelerated graphics following the couple step instructions if you would like

**Comment by smartavionics** [comment]
Hmm, the 18.04 images are not for the pi 4 so I can't use them.

**Comment by theofficialgman** [comment]
@smartavionics sorry its not directly on that page. thats just documentation, you can get to the image for the pi4 from that page but its not direct.
you can find the 18.04 rpi4 image here http://cdimage.ubuntu.com/ubuntu/releases/bionic/release/

**Comment by smartavionics** [comment]
OK, thanks. I'll look at doing that when I can. Quite busy at the moment so I can't say when it will get done.
